src/rsz/src/MakeResizer.cc

```diff
@@ -63,7 +63,8 @@ initResizer(OpenRoad *openroad)
                                gui::Gui::get(),
                                openroad->getDb(),
                                openroad->getSta(),
-                               openroad->getGlobalRouter());
+                               openroad->getGlobalRouter(),
```

Comment:
the global router is no longer needed since you moved the only functions it accessed